frontend/src/test/java/com/google/sps/InMemoryPlaceVisitStorageTest.java

```diff
@@ -0,0 +1,333 @@
+package com.google.sps;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth8.assertThat;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.util.List;
+import com.google.tripmeout.frontend.storage.InMemoryPlaceVisitStorage;
+import com.google.tripmeout.frontend.PlaceVisitModel;
+import com.google.tripmeout.frontend.error.PlaceVisitAlreadyExistsException;
+import com.google.tripmeout.frontend.error.PlaceVisitNotFoundException;
+
+@RunWith(JUnit4.class)
+public final class InMemoryPlaceVisitStorageTest {
+  private static final InMemoryPlaceVisitStorage NO_PLACE_VISITS = new InMemoryPlaceVisitStorage();
```

Comment:
not super sure what the point of this is over just calling `new InMemoryPlaceVisitStorage()` inline is.